web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -10,6 +10,24 @@
       />
     </v-toolbar>
     <v-container
+      v-if="dandisetDoesNotExist"
+      class="d-flex justify-center align-center"
+      style="height: 50vh;"
+    >
+      <div class="d-block">
+        <span class="text-h5">
+          <v-icon>mdi-alert</v-icon>
+          Error: Dandiset does not exist.
+        </span>
+        <br><br>
+        <span class="text-body-2">
+          Proceed back to the <a href="/dandiset">Public Dandisets page</a>
```

Comment:
```suggestion
          Proceed to the <a href="/dandiset">Public Dandisets page</a>
```

since they may not have come from the public dandisets page.